tethys_apps/utilities.py

```diff
@@ -620,13 +621,18 @@ def get_configured_standalone_app():
     from tethys_apps.models import TethysApp
 
     standalone_app = settings.STANDALONE_APP
+    app = None
 
-    if standalone_app:
-        app = TethysApp.objects.get(package=standalone_app)
-    else:
-        app = TethysApp.objects.first()
-        if not app:
-            raise ObjectDoesNotExist("No Tethys Apps have been installed")
+    try:
+        if standalone_app:
+            app = TethysApp.objects.get(package=standalone_app)
+        else:
+            app = TethysApp.objects.first()
```

Comment:
What happens if the database is configured, but there is no app installed? Do these raise an exception? We need to catch those exceptions too.